common/lib/xmodule/xmodule/js/src/problem/edit.coffee

```diff
@@ -433,42 +433,86 @@ class @MarkdownEditingDescriptor extends XModule.Descriptor
           // Line split here, trim off leading xxx= in each function
           var answersList = p.split('\n'),
 
-              processNumericalResponse = function (value) {
-                  // Numeric case is just a plain leading = with a single answer
-                  value = value.replace(/^\=\s*/, '');
-                  var params, answer, string;
+              processNumericalResponse = function (values) {
+                  var values, params, answer, firstAnswer, string, additionalAnswerString;
 
-                  var textHint = extractHint(value);
+                  isRangeToleranceCase = function(answer) {
+                      if (_.contains([ '[', '(' ], answer[0]) && _.contains([ ']', ')' ], answer[answer.length-1]) ) {
```

Comment:
We can skip if here, how about this:
`return _.contains([ '[', '(' ], answer[0]) && _.contains([ ']', ')' ], answer[answer.length-1])`